docs/spark/how-to-guides/hdinsight-notebook-installation.md

```diff
@@ -0,0 +1,123 @@
+description: Learn how to install .NET for Apache Spark Interactive Notebooks on Azure HDInsight
+ms.date: 02/18/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Install .NET for Apache Spark Interactive Notebooks on Azure HDInsight
+
+This how-to provides the steps to install .NET for Apache Spark interactive notebooks on Azure HDInsight Spark Clusters.
+
+## Background
+
+To provide the ability to use .NET for Apache Spark within your Jupyter Notebooks, an important pre-requisite is a C# REPL that is capable of executing your C# code line-by-line (and preserving execution state, where needed). We have integrated [dotnet-try](https://github.com/dotnet/try) as our official dotnet REPL to provide you with the best experience.
+
+To provide the end-to-end integration, we had to add a Jupyter kernel along with necessary modifications to [Apache Livy](https://github.com/apache/incubator-livy) and [sparkmagic](https://github.com/jupyter-incubator/sparkmagic). Since these code changes have not yet been merged into the respective open source projects, to be able to get the out-of-box .NET for Apache Spark experience, you will have to patch components in your HDInsight Spark Cluster with the binaries we provide.
+
+To enable .NET for Apache Spark through the Jupyter Notebook Experience, you would have to follow a few manual steps through [Ambari](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-manage-ambari) and submit [script actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux) on HDInsight Spark cluster.
+
+>**Disclaimer:** Please note that this is *experimental* and not supported by the HDInsight Spark team. *Use it at your own risk*. Please do remember that we are working hard to get these changes into the respective projects so you do not need these manual steps.
+
+## Pre-requisites:
+Create an [Azure HDInsight Spark](https://docs.microsoft.com/en-us/azure/hdinsight/spark/apache-spark-jupyter-spark-sql-use-portal#create-an-hdinsight-spark-cluster) cluster (by visiting the **[Azure Portal](https;//portal.azure.com)**, selecting **+ Create a Resource**, and creating a new Azure HDInsight cluster resource), select the **Overview** tab on your cluster. Ensure you have created a cluster with **Spark 2.4** and **HDI 4.0**.
+
+## Installation Procedure
+Open the [Azure portal](https://ms.portal.azure.com/#home). Select **HDInsight Spark cluster** you created in the previous step.
+### Step 1. Stop Livy Server
+1.1 From the portal, select **Overview**, and then select **Ambari home**. If prompted, enter the cluster login credentials for the cluster.
+
+<img src="../media/hdi-spark-notebooks/select-ambari.png" alt="StopLivyServerImage" width="800"/>
+
+1.2 Select **Spark2**, and then select **LIVY FOR SPARK2 SERVER**.
+
+<img src="../media/hdi-spark-notebooks/select-livyserver.png" alt="StopLivyServerImage" width="800"/>
```

Comment:
This image doesn't render in preview: https://review.docs.microsoft.com/en-us/dotnet/spark/how-to-guides/hdinsight-notebook-installation?branch=pr-en-us-17119#installation-procedure

Reply:
It's get rendered now.